modules/subscriptions.php

```diff
@@ -371,6 +399,63 @@ function subscription_comment_subscribe_setting() {
 	<?php
 	}
 
+	/**
+	 * Someone follows my blog section
+	 *
+	 * @since 8.1
+	 */
+	public function social_notifications_subscribe_section() {
+		// Atypical usage here. We emit jquery to move subscribe notification checkbox to be with the rest of the email notification settings
+		?>
+		<script type="text/javascript">
+			jQuery( function( $ )  {
+				var table = $( '#social_notifications_subscribe' ).parents( 'table:first' ),
+					header = table.prevAll( 'h2:first' ),
+					newParent = $( '#moderation_notify' ).parent( 'label' ).parent();
+
+				if ( !table.length || !header.length || !newParent.length ) {
+					return;
+				}
+
+				newParent.append( '<br/>' ).append( table.end().parent( 'label' ).siblings().andSelf() );
+				header.remove();
+				table.remove();
+			} );
+		</script>
+		<?php
+	}
+
+	/**
+	 * Someone follows my blog Toggle
+	 *
+	 * @since 8.1
+	 */
+	function social_notifications_subscribe_field() {
+		$checked = intval ('on' == get_option( 'social_notifications_subscribe', 'on' ) ); ?>
+
+		<label>
+			<input type="checkbox" name="social_notifications_subscribe" id="social_notifications_subscribe" value="1" <?php checked( $checked ); ?> />
+			<?php _e( 'Someone follows my blog	', 'jetpack' ); ?>
```

Comment:
> Why is there a tab symbol inside the translatable string after blog? If the label needs additional markup, let's move it outside of the i18n?

This was my mistake, my next commit will remove this tab symbol.

> Also this string really needs a translator comment, otherwise it's not clear what this string is for. Let's add something like:

Gotcha! This makes sense.